### DISCUSSION TITLE
Eduponics Mini v2.0 is launching on Crowd Supply!

### DISCUSSION THREAD

**Discussion by Zontex** [STEMinds/STEMinds-Docs#10] [Announcements]
Hey folks!

I am thrilled to announce we are launching Eduponics v2.0 on Crowd Supply with a brand new extension board - the 4-channel valve extension board.

Please feel free to subscribe and spread the word :)

Check the link below: https://www.crowdsupply.com/steminds/eduponics-mini-v2

**Comment by GalGreenfield** [comment]
That's awesome! I have the original Eduponics and it's awesome!
Best of luck with the new campaign!
